### PR TITLE
Add space at the bottom of the deployment flow for the cookie notice

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes: #3348.

### CODE REVIEW

**Comment by huwshimi** [comment]
[Fix cookie notice covering the deploy button](https://trello.com/c/OEWCnXEF/68-fix-cookie-notice-covering-the-deploy-button)

**Comment by huwshimi** [comment]
QA:
- start guiproxy with GTM_enabled e.g. guiproxy -env staging -config '{"GTM_enabled": true}'
- open the gui in a private browser and log in
- add an app
- open the deployment flow
- scroll to the bottom and check that the cookie notice is not over the deploy button

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8292/

File: jujugui/static/gui/src/app/init/cookie-util.js [hunk 1]
```diff
@@ -12,14 +12,23 @@ const Notification = require('../components/notification/notification');
...
     _removeNotification(doc);
   }
 }
 
+/**
+  Check whether the cookie notice should be visible.
+  @param {Object} doc Reference to the window.document.
+  @returns {Boolean} Whether the notification should be showed.
+*/
+function show_notification(doc) {
```
Comment (hatched) [inline jujugui/static/gui/src/app/init/cookie-util.js]: `shouldShowNotification` ?

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8307/

**Comment by hatched** [comment]
Thanks @huwshimi smart implementation.

:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**